io.github.ad_on_is.Resticity.yml

```diff
@@ -0,0 +1,76 @@
+id: io.github.ad_on_is.Resticity
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+
+
+command: resticity
+finish-args:
+  - --share=ipc # Share IPC namespace with the host (necessary for X11).
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri # OpenGL rendering support.
+  - --talk-name=org.freedesktop.Notifications
+  - --filesystem=home
+  - --filesystem=host
+  - --share=network
+  - --talk-name=org.kde.StatusNotifierWatcher
+
+modules:
+  
+  - name: rclone
+    buildsystem: simple
+    build-commands:
+      - mkdir -p $FLATPAK_DEST/bin
+      - install -Dm755 rclone $FLATPAK_DEST/bin/rclone  
+      - chmod a+x $FLATPAK_DEST/bin/rclone
+    sources:
+      - type: archive
+        only-arches: [x86_64]
+        url: https://downloads.rclone.org/v1.66.0/rclone-v1.66.0-linux-amd64.zip
+        sha256: b4d304b1dc76001b1d3bb820ae8d1ae60a072afbd3296be904a3ee00b3d4fab9
+      - type: archive
+        only-arches: [aarch64]
+        url: https://downloads.rclone.org/v1.66.0/rclone-v1.66.0-linux-arm64.zip
+        sha256: c50a3ab93082f21788f9244393b19f2426edeeb896eec2e3e05ffb2e8727e075
+  - name: restic
+    buildsystem: simple
+    build-commands:
+      - mkdir -p $FLATPAK_DEST/bin
+      - bzcat restic_*.bz2 > $FLATPAK_DEST/bin/restic
```

Comment:
See was ignored: https://github.com/flathub/flathub/pull/5074#discussion_r1534539459

Reply:
I tried using archive, but I get the error, that bz2 is not supported.